src/Core_Command.php

```diff
@@ -636,7 +639,9 @@ private function _multisite_convert( $assoc_args ) {
 EOT;
 
 			$wp_config_path = Utils\locate_wp_config();
-			if ( is_writable( $wp_config_path ) && self::modify_wp_config( $ms_config ) ) {
+			if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-config' ) ) {
+				WP_CLI::log( "Addition of multisite constants to 'wp-config.php' skipped. You need to add them manually:" . PHP_EOL );
```

Comment:
Or, we could include the constants similar to below ` You need to add them manually:" . PHP_EOL . $ms_config`